big_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -582,6 +582,9 @@ test_pm_with_graceful_reconnection_to_different_server(Config) ->
     escalus_users:delete_users(Config, [{eve, [{port, 5222} | EveSpec]}]).
 
 test_pm_with_ungraceful_reconnection_to_different_server(Config0) ->
+%% This tests the feature which has not been implemented (yet?) by mod_global_distrib
```

Comment:
This comment is no longer relevant as it turned out that this case is OK, however still susceptible to a *very* unlikely race condition discussed earlier (which wasn't responsible for the failures). So it shouldn't be turned off or removed.

Reply:
Now the relevant comment is included